tests/test_model.py

```diff
@@ -189,7 +189,9 @@ def test_process_tsv(self):
 		try:
 			self.assertEqual(expect, man_made)
 		except:
+			# Might have a subclass from an extension
 			expect[u'subs'] = []
+			man_made[u'subs'] = []
```

Comment:
If this fallback is always available, and there is no indication that it was used to make the assertion pass, why not just delete the `subs` keys before the call to `assertEqual` and avoid the whole `try`/`except` block?

Reply:
:+1: done!